account_banking_pain_base/models/payment_mode.py

```diff
@@ -14,3 +15,35 @@ class PaymentMode(models.Model):
         help="If active, Odoo will convert each accented caracter to "
         "the corresponding unaccented caracter, so that only ASCII "
         "caracters are used in the generated PAIN file.")
+    initiating_party_issuer = fields.Char(
+        string='Initiating Party Issuer', size=35,
+        help="This will be used as the 'Initiating Party Issuer' in the "
+        "PAIN files generated by Odoo. If not defined, Initiating Party "
+        "Issuer from company will be used.\n"
+        "Common format (13): \n"
+        "- Country code (2, optional)\n"
+        "- Company idenfier (N, VAT)\n"
+        "- Service suffix (N, issued by bank)")
+    initiating_party_identifier = fields.Char(
+        string='Initiating Party Identifier', size=35,
+        help="This will be used as the 'Initiating Party Identifier' in "
+        "the PAIN files generated by Odoo. If not defined, Initiating Party "
+        "Identifier from company will be used.\n"
+        "Common format (13): \n"
+        "- Country code (2, optional)\n"
+        "- Company idenfier (N, VAT)\n"
+        "- Service suffix (N, issued by bank)")
+    sepa_type = fields.Char(compute="_compute_sepa_type")
+
+    def _sepa_type_get(self):
```

Comment:
Well, for not doing the loop again in all inherited methods. Just get the type.
The api.depends one take care about multi record and fields dependency